README.md

```diff
@@ -48,11 +48,17 @@ var childWalletSecret = lib.derivePublic(parentWalletPublicKey, 1, 1)
 * `Buffer toPublic(Buffer privateKey)`
 * `Buffer decodePaperWalletMnemonic(string paperWalletMnemonic)`
 * `Buffer xpubToHdPassphrase(Buffer xpub)`
-* `string packAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer deriveBootstrapAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer deriveBaseAddress(Buffer pubKey, Buffer stakePubKey, int addressTypeId, int protocolMagic, Bool isStakeHash = false)`
+* `Buffer derivePointerAddress(Buffer pubKey, Object pointer, int addressTypeId, int protocolMagic)`
+* `Buffer deriveEnterpriseAddress(Buffer pubKey, int addressTypeId, int protocolMagic)`
```

Comment:
note for reworking -> protocol magic and network id indeed are not the same thing as I just found out discussing it with Gabriel. NetworkId is a new concept introduced in Shelley and it would be only 4 bits whereas procolMagic is 4bytes and is "baked" into Byron addresses only for testnet